cnxepub/models.py

```diff
@@ -128,12 +128,12 @@ def model_to_tree(model, title=None, lucent_id=TRANSLUCENT_BINDER_ID):
         {'id': <id>|'subcol', 'title': <title>, 'contents': [<tree>, ...]}
 
     """
-    if type(model) is TranslucentBinder:
+    id = model.ident_hash
+    if id is None and isinstance(model, TranslucentBinder):
```

Comment:
eventually, all binders and sub-binders will have Ids so, this won't matter. ARe you suggesting we should allow unidentified binders out to pass, with id=None ?